Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -160,45 +160,51 @@ def __init__(self):
 class RuleDescriptor(SimpleDescriptor):
     """<!-- optional: list of substitution rules -->
     <rules>
-        <rule name="vertical.bars" enabled="true">
```

Comment:
Can we have an optional `enabled` in the rule element?

Reply:
sure. I removed it from the example because it was not implemented nor documented

Reply:
Fine, remove it :)